pkg/apis/apps/validation/validation_test.go

```diff
@@ -2720,7 +2753,7 @@ func TestValidateDaemonSet(t *testing.T) {
 	}
 }
 
-func validDeployment() *apps.Deployment {
+func validDeployment(expectedErrNum int, enableSkipReadOnlyValidationGCE bool) *apps.Deployment {
```

Comment:
I'm confiused - these are not used anywhere?

Reply:
I didn't find a deployment similar testing :(